### ISSUE TITLE
"No Depth Perception" Page reference, in Basic Set Advantage Library, listed as 144 instead of 145. 

### ISSUE THREAD

**Issue by Bootrot** [richardwilkes/gcs#668]
Hopefully this is the correct place to report a minor typo, I don't know if I would actually call it a **_BUG_**?

**Comment by richardwilkes** [comment]
Yep, this is fine. Thanks. Had I noticed it just slightly earlier, I would have included it in the update I just released. Will have to wait for next time.
